cabal.project

```diff
@@ -41,3 +41,13 @@ benchmarks: True
 
 -- Always show detailed output for tests
 test-show-details: direct
+
+source-repository-package
+  type: git
+  location: https://github.com/IntersectMBO/cardano-node-emulator
+  tag: 20f3faf2ef9ca97edcdb7b037507b35a5d76673b
+  --sha256: sha256-bhaJsWqjhojpbwxLX1+vVPSPz86tbhDcBUHk6M1HPzU=
+  subdir:
+    plutus-script-utils
+    plutus-ledger
```

Comment:
Can't we use the packages from CHaP?

Reply:
It isn't merged yet because it's a monorepo that would require fixes to cardano-node-enulator as well.

Reply:
Have raised https://github.com/IntersectMBO/cardano-node-emulator/issues/38

Reply:
Do we need the `plutus-ledger` entry here? I don't see it used in our cabal files.

Reply:
This dependency will not help us much anymore if we switch to aiken. We should close this.